app/controllers/support_interface/survey_emails_controller.rb

```diff
@@ -0,0 +1,26 @@
+module SupportInterface
+  class SurveyEmailsController < SupportInterfaceController
+    before_action :set_application_form
+
+    def show; end
+
+    def deliver
+      CandidateMailer.survey_email(@application_form).deliver
+
+      candidate_email = @application_form.candidate.email_address
+      audit_comment = I18n.t('survey_emails.send.audit_comment', candidate_email: candidate_email)
+      application_comment = SupportInterface::ApplicationCommentForm.new(comment: audit_comment)
+      application_comment.save(@application_form)
```

Comment:
The `SendChaseEmailToRefereeAndCandidate` encapsulates code that is quite similar to this, and by being a service, it benefits from being easy to unit test. Do we think it's worth refactoring this to also use a service?

Reply:
Yes! I thought about doing this but I thought it might be better to leave it for now and do it when we add in the sending a survey email to a referee. What do you reckon? Now or later?

Reply:
I'm OK with later :)